boards/trellis_m4/src/lib.rs

```diff
@@ -212,25 +220,3 @@ impl Keypad {
         })
     }
 }
-
-/// Convenience for setting up the labelled SDA, SCL pins to
-/// operate as an I2C master running at the specified frequency.
-pub fn i2c_master<F: Into<Hertz>>(
-    clocks: &mut GenericClockController,
-    bus_speed: F,
-    sercom4: SERCOM4,
-    mclk: &mut MCLK,
-    sda: gpio::Pb8<Input<Floating>>,
-    scl: gpio::Pb9<Input<Floating>>,
-    port: &mut Port,
-) -> I2CMaster4 {
-    let gclk0 = clocks.gclk0();
-    I2CMaster4::new(
-        &clocks.sercom4_core(&gclk0).unwrap(),
-        bus_speed.into(),
-        sercom4,
-        mclk,
-        sda.into_pad(port),
-        scl.into_pad(port),
-    )
-}
```

Comment:
With multiple I2C devices I thought it'd make sense to move this into the corresponding pins type. It certainly helps shave off a few arguments!

Reply:
Could you please re-export the new i2c_master function in it's place?

Reply:
Done